resources/build/Makefile

```diff
@@ -0,0 +1,63 @@
+default: docker-image
```

Comment:
[fix] This Makefile was introduced as a blank file in the first commit

Reply:
Addressed in rebase.